rhdm70-image-streams.yaml

```diff
@@ -22,9 +23,6 @@ items:
         tags: rhdm,xpaas
         supports: rhdm:7.0,xpaas:1.4
         version: '1.0'
-      from:
```

Comment:
@spolti OK, I am fine with put the insecure option in docs. What about the dockerImageRepository setting?
 Do you manage to reproduce it on the ER1 product build? It seems to me that the "from"
won't work with dev registry.